### DISCUSSION TITLE
How to make Lunarvim switch buffers with `<leader>NUMBER`

### DISCUSSION THREAD

**Discussion by ghost** [LunarVim/LunarVim#4058] [Q&A] [answered]
Currently, I can switch buffers with `<leader>bb` `<leader>bn` `<leader>bj`, but these operations are so verbose.
I whish I can switch buffer like `<leader>1` `<leader>5` etc...

**Comment by brneor** [comment, marked as answer]
You can map `<leader> NUMBER` to command `:BufferLineGoToBuffer NUMBER`. Not sure if there's a smarter way to do this or if you have to hardcode the numbers in mappings.

**Reply by ghost** [reply]
tank you for your answer
